components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Gateway/src/Server/GatewayServer.cs

```diff
@@ -1731,7 +1732,7 @@ private class DummyDialog : IApplicationMessageDlg {
                 DiscoveryUrls = null
             };
             endpoints = _endpoints;
-            return new List<Task> { Task.CompletedTask };
+            return new List<ServiceHost>(); // TODO: We should probably create a service host here - but is this code really used?
         }
```

Comment:
>} [](http://example.com/codeflow?start=8&length=1)

implement /// <summary>
        /// Creates an instance of the service host.
        /// </summary>
        public virtual ServiceHost CreateServiceHost(ServerBase server, params Uri[] addresses)
        {
            return new ServiceHost()
        }